py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py

```diff
@@ -79,6 +79,7 @@ def __init__(
         name: str = "",
         settings: CompilationSettings = CompilationSettings(),  # Assumes engine was built with default compilation settings if object not passed
         weight_name_map: Optional[dict[Any, Any]] = None,
+        graph_module: torch.fx.GraphModule = None,
```

Comment:
@narendasan I tried to do refitting like in Python runtime but didn't work. Any suggestions? should I do in C++ or Python?

Reply:
Doesnt refit already work on both apis?

Reply:
Also why do we need the graph module in this module?

Reply:
1. In this PR I moved the refitting part into TRTModule, so only works for Python runtime.

2. graph module is used for refitting